### PR TITLE
[#126549207] As an Engineer, so I can upgrade warehouse to Rails 5, **create a new Redshift adapter for rails 5**

### PR DESCRIPTION
**Pull request by nfoen**
@RoryO , @cobbr2 please CR

[tracker](https://www.pivotaltracker.com/n/projects/1613883/stories/126549207)

### CODE REVIEW

File: activerecord5-redshift-adapter.gemspec [hunk 1]
```diff
@@ -0,0 +1,19 @@
+Gem::Specification.new do |s|
+  s.platform = Gem::Platform::RUBY
+  s.name = 'activerecord5-redshift-adapter'
+  s.version = '0.0.1'
+  s.summary = 'Amazon Redshift adapter for ActiveRecord '
+  s.description = 'Amazon Redshift _makeshift_ adapter for ActiveRecord 5.'
+  s.license = 'MIT'
+
+  s.author = ['Nancy Foen']
```
Comment (RoryO) [inline activerecord5-redshift-adapter.gemspec]: should add the original author to this array as well

File: activerecord5-redshift-adapter.gemspec [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  s.email = 'nancy.foen@grandrounds.com'
+  s.homepage = 'https://github.com/ConsultingMD/activerecord5-redshift-adapter'
+
+  s.files = Dir.glob(['LICENSE', 'README.md', 'lib/**/*.rb'])
+  s.require_path = 'lib'
+
+  s.required_ruby_version = '>= 2.2.2'
+  s.add_dependency 'pg'
```
Comment (RoryO) [inline activerecord5-redshift-adapter.gemspec]: was there a minimum version of pg for this to work?

File: lib/active_record/connection_adapters/redshift/schema_definitions.rb [hunk 3]
```diff
@@ -2,21 +2,6 @@ module ActiveRecord
   module ConnectionAdapters
```
Comment (RoryO) [inline lib/active_record/connection_adapters/redshift/schema_definitions.rb:2]: It doesn't look like this file changed, just moved around?

**Comment by RoryO** [comment]
You know more about AR5 than me at this point. I haven't had a chance to look at it.

File: lib/active_record/connection_adapters/redshift/schema_definitions.rb (hunk 3 above)
Comment (nfoen) [inline lib/active_record/connection_adapters/redshift/schema_definitions.rb:2]: We moved the primary_key method from the TableDefinition class into the ColumnMethods module. Since they are all defined here is quite hard to follow.

File: README.md [hunk 4]
```diff
@@ -1,36 +1,34 @@
...
-For Rails 4.2, write following in Gemfile:
-```
-gem 'activerecord4-redshift-adapter', '~> 0.2.0'
-```
-For Rails 4.1:
-```
-gem 'activerecord4-redshift-adapter', '~> 0.1.1'
+For Rails 5, write following in Gemfile:
 ```
+gem 'activerecord5-redshift-adapter'
```
Comment (nfoen) [inline README.md:22]: @RoryO are we publishing this gem, or should we provide the git path instead?

Comment (RoryO) [inline README.md:22]: Should publish it in the off chance that someone stumbles upon it and contributes.

**Comment by nfoen** [comment]
@RoryO , @cobbr2 any other comments on this?  I would like this to get merged soon to be able to upgrade warehouse to rails 5

**Comment by RoryO** [comment]
I'll do it!
